spec/factories/profiles.rb

```diff
@@ -53,7 +48,7 @@
     end
 
     trait :fraud_rejection do
-      fraud_pending_reason { 'threatmetrix_review' }
+      fraud_pending_reason { 'threatmetrix_reject' }
```

Comment:
!

Reply:
This code is ok. A user can get a response of threatmetrix_review from ThreatMetrix, and then our fraud team can decide to reject them.